tests/integrationv2/test_key_update.py

```diff
@@ -0,0 +1,138 @@
+import copy
+import pytest
+
+from configuration import available_ports, Ciphers
+from common import ProviderOptions, Protocols, data_bytes
+from fixtures import managed_process
+from providers import Provider, S2N, OpenSSL
+from utils import invalid_test_parameters
+
+
+@pytest.mark.uncollect_if(func=invalid_test_parameters)
+# These are the only ciphers that utilize the key update feature
+@pytest.mark.parametrize(
+    "cipher",
+    [
+        Ciphers.AES128_GCM_SHA256,
+        Ciphers.AES256_GCM_SHA384,
+        Ciphers.CHACHA20_POLY1305_SHA256,
```

Comment:
Put this list in a common place. Also, shouldn't this just be "all TLS13 ciphers" instead of a special list?